controller/api/public/grpc_server.go

```diff
@@ -472,6 +473,11 @@ func formatQuery(query string, req *pb.MetricRequest, sumBy string) (string, err
 		}
 	}
 
+	for _, exclusion := range filterExclusions {
+		filterLabels = append(filterLabels, fmt.Sprintf("%s!~\"%s\"", targetDeployLabel, fmt.Sprintf("%s/.*", exclusion)))
+		filterLabels = append(filterLabels, fmt.Sprintf("%s!~\"%s\"", sourceDeployLabel, fmt.Sprintf("%s/.*", exclusion)))
```

Comment:
consider avoiding the nested `Sprintf`'s:
```go
fmt.Sprintf("%s!~\"%s/.*\"", targetDeployLabel, exclusion)
```

also, this yields filters like:
```
targetDeployLabel!~"exclusion/.*"
```
is there any way to filter by exact match (`!=`)? or, to put it another way, is it possible this could filter out things we didn't want to filter out?